library/IReplicatedTableEntity.cs

```diff
@@ -50,6 +51,10 @@ public interface IReplicatedTableEntity : ITableEntity
         /// New clients can use their own timeout (LockTimeout) value to determine whether to initiate replication or not.
         /// </summary>
         DateTimeOffset _rtable_LockAcquisition { get; set; }
+
+        void ReadEntity(IDictionary<string, object> properties);
```

Comment:
do we need to add this function to the interface ? This class seems to be defined only to extend to store more data

Reply:
ReadEntity is referenced in ReplicatedTable.ConvertToIReplicatedTableEntity() to go from TableEntity to IReplicatedTableEntity.
WriteEntity is referenced in ReplicatedTable.ToTableEntity() to convert IReplicatedTableEntity to TableEntity.